v-next/example-project/test/mocha/other-mocha-test.ts

```diff
@@ -0,0 +1,16 @@
+import assert from "node:assert/strict";
+import { describe, it } from "mocha";
+
+// import hre from "@ignored/hardhat-vnext";
+
+// describe("Other mocha test", () => {
+//   it("should have the example task", () => {
+//     assert.ok(hre.tasks.getTask("example"));
+//   });
+// });
+
+describe("Other mocha test", () => {
+  it("should have the example task", () => {
+    assert.ok(true);
+  });
+});
```

Comment:
We should uncomment the version that uses `hre`, I think. 

```suggestion
import hre from "@ignored/hardhat-vnext";

describe("Other mocha test", () => {
  it("should have the example task", () => {
    assert.ok(hre.tasks.getTask("example"));
  });
});
```